qemu/tests/virtio_fs_multi_users_access.py

```diff
@@ -0,0 +1,167 @@
+from virttest import error_context, utils_test, utils_disk, utils_misc
+
+from provider import win_driver_installer_test, virtio_fs_utils
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Different users access the shared directory.
+
+    1) Create a shared directory for testing on the host.
+    2) Run the virtiofsd daemon on the host.
+    3) Boot a guest on the host.
+    4) Log into guest and create the users if there is NOT dns. Otherwise,
```

Comment:
Suggest to update with:
4) Log into guest and create a new user. For AD account test, please join in the AD server.